README.md

```diff
@@ -30,6 +30,8 @@ If you are allowing user to capture video add `NSCameraUsageDescription` add `NS
 
 No permissions required (`saveToPhotos` requires permission [check](#note-on-file-storage)).
 
+Note: This library does not require Manifest.permission.CAMERA, if your app declares as using this permission in manifest then you have to obtain the permission before using `launchCamera`.
```

Comment:
Consider adding more emphasis about the whole library not requiring additional permissions.
As this comes up a lot in the issues.

Reply:
Above this line I have mentioned `No permissions required`. So I am out of ideas.

Reply:
Lets see how the issues come then we will update appropriately. When in doubt better to avoid adding more text.